quack_app/lib/components/navbar.dart

```diff
@@ -19,10 +20,7 @@ class _NavBarState extends State<NavBar> {
   // To implement the new page,
   // change the Text() below to the page's class (i.e. HomeScreen())
   static const List<Widget> _widgetOptions = <Widget>[
-    Text(
-      'Index 0: Settings',
-      style: optionStyle,
-    ),
+    SettingsScreen(title: "Settings"),
```

Comment:
I feel like settings should be the last one (the rightmost one)

Reply:
@MimiOlayeye, what do you think?